src/pyhf/infer/calculators.py

```diff
@@ -527,7 +558,7 @@ def distributions(self, poi_test, track_progress=None):
         bkg_pdf = self.pdf.make_pdf(tensorlib.astensor(bkg_pars))
         bkg_sample = bkg_pdf.sample(sample_shape)
 
-        teststat_func = qmu_tilde if self.qtilde else qmu
+        teststat_func = test_statistics.get(self.test_stat)
```

Comment:
Only applies if people agree on the API comment
```suggestion
        teststat_func = test_statistics.get_test_stat(self.test_stat)
```